pages/dkp/konvoy/2.2/choose-infrastructure/pre-provisioned/prerequisites-airgapped/index.md

```diff
@@ -0,0 +1,158 @@
+    curl --output artifacts/pip-packages.tar.gz -O https://downloads.d2iq.com/dkp/airgapped/pip-packages/pip-packages.tar.gz
+    ```
+
+1.  Export the following environment variables, ensuring that all control plane and worker nodes are included:
+
+    ```bash
+    export CONTROL_PLANE_1_ADDRESS="<control-plane-address-1>"
+    export CONTROL_PLANE_2_ADDRESS="<control-plane-address-2>"
+    export CONTROL_PLANE_3_ADDRESS="<control-plane-address-3>"
+    export WORKER_1_ADDRESS="<worker-address-1>"
+    export WORKER_2_ADDRESS="<worker-address-2>"
+    export WORKER_3_ADDRESS="<worker-address-3>"
+    export WORKER_4_ADDRESS="<worker-address-4>"
+    export SSH_USER="<ssh-user>"
+    export SSH_PRIVATE_KEY_FILE="<private key file>"
+    ```
+
+    `SSH_PRIVATE_KEY_FILE` must be either the name of the SSH private key file in your working directory or an absolute path to the file in your user's home directory.
+
+1.  Generate an `inventory.yaml` to be used with `konvoy-image upload` in the next step:
+
+    ```yaml
+    cat <<EOF > inventory.yaml
+    all:
+      vars:
+        ansible_user: $SSH_USER
+        ansible_port: 22
+        ansible_ssh_private_key_file: $SSH_PRIVATE_KEY_FILE
+      hosts:
+        $CONTROL_PLANE_1_ADDRESS:
+          ansible_host: $CONTROL_PLANE_1_ADDRESS
+        $CONTROL_PLANE_2_ADDRESS:
+          ansible_host: $CONTROL_PLANE_2_ADDRESS
+        $CONTROL_PLANE_2_ADDRESS:
+          ansible_host: $CONTROL_PLANE_3_ADDRESS
+        $WORKER_1_ADDRESS:
+          ansible_host: $WORKER_1_ADDRESS
+        $WORKER_2_ADDRESS:
+          ansible_host: $WORKER_2_ADDRESS
+        $WORKER_3_ADDRESS:
+          ansible_host: $WORKER_3_ADDRESS
+        $WORKER_4_ADDRESS:
+          ansible_host: $WORKER_4_ADDRESS
+    EOF
+    ```
+
+1.  Copy the artifacts onto cluster hosts:
+
+    ```bash
+    konvoy-image upload artifacts --container-images-dir=./artifacts/images/ --os-packages-bundle=./artifacts/"$VERSION"_"$BUNDLE_OS".tar.gz --pip-packages-bundle=./artifacts/pip-packages.tar.gz
+    ```
+
+## Seed your docker registry
+
+Before creating a Kubernetes cluster you must have the required images in a local docker registry. This registry must be accessible from both the bastion machine and the machines that will be created for the Kubernetes cluster.
+
+
+1.  Download the images bundle:
+
+    ```bash
+    curl -o konvoy-image-bundle.tar.gz -O downloads.d2iq.com/dkp/v2.2.0/konvoy_image_bundle_v2.2.0_linux_amd64.tar.gz
+    ```
+
+1.  Place the bundle in a location where you can load and push the images to your private docker registry.
+
+1.  Set an environment variable with your registry address:
+
+    ```bash
+    export DOCKER_REGISTRY_ADDRESS=<https/http>://<registry-address>:<registry-port>
+    ```
+
+1.  Run the following command to load the air-gapped image bundle into your private Docker registry:
+
+    ```bash
+    dkp push image-bundle --image-bundle konvoy-image-bundle.tar.gz --to-registry <$DOCKER_REGISTRY_ADDRESS>
```

Comment:
We need to remove the `<>`:
`<$DOCKER_REGISTRY_ADDRESS>` -> `$DOCKER_REGISTRY_ADDRESS`.

Reply:
:(